lib/nfs_common.pm

```diff
@@ -125,10 +125,10 @@ sub client_common_tests {
     assert_script_run 'sed -i \'/nfs/d\' /etc/fstab';
 
     # compare saved and current fstab, should be same
-    assert_script_run 'diff -b /etc/fstab fstab_before';
+    #    assert_script_run 'diff -b /etc/fstab fstab_before';
 
     # compare last line, should be not deleted
-    assert_script_run 'diff -b <(tail -n1 /etc/fstab) <(tail -n1 fstab_before)';
+    #assert_script_run 'diff -b <(tail -n1 /etc/fstab) <(tail -n1 fstab_before)';
```

Comment:
This part was also failing http://kepler.suse.cz/tests/8773#step/yast2_nfs4_client/58. IMHO it seems the check is a bit strict

Reply:
`/etc/fstab` is being updated by YaST2. Unfortunately, YaST2 strips the `default` mount option that is among the original entries. It also aligns the records into a table format.

Reply:
So this is expected, that in `yast2-storage-ng || libstorage-ng` modifies the content of `/etc/fstab`.

Here is a brief explain from YaST2 dev:

It simply uses "defaults" as the placeholder for "no specific options" and it omits "defaults" as soon as there is any other parameter, no matter what it is